src/core/address.h

```diff
@@ -10,6 +10,7 @@
 // the length of an address
 #define IOTA_ADDRESS_BYTES (1 + ED_PUBLIC_KEY_BYTES)  // address version + address data
 
+// W-OTS won't be supported anymore, we should remove this
```

Comment:
oh missed this one, it's part of RFC. it will be changed once the RFC is updated.